etc/env.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 source "/scratch3/BMC/gsd-hpcs/Daniel.Abdi/miniconda3/etc/profile.d/conda.sh"
 conda activate hrrrcast-live
```

Comment:
This need to be changed to `hrrrcast` for both `env.sh` and `env.csh`

Reply:
Updated.